manage.py

```diff
@@ -18,8 +18,8 @@
 manager.add_option(
     '-m', '--cfgmode', dest='config_mode', default='Development')
 manager.add_option('-f', '--cfgfile', dest='config_file', type=p.abspath)
-manager.add_command('runserver', Server(port=c.PORT))
-manager.add_command('serve', Server(port=c.PORT))
+manager.add_command('runserver', Server(host=c.HOST, port=c.PORT))
+manager.add_command('serve', Server(host=c.HOST, port=c.PORT))
 manager.main = manager.run
```

Comment:
This may not be needed since I think the HOST env is used by default. Ok, to keep for now until I can confirm.